# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoaderBase.h
# Commit: e2701bd43ce2
# Full Hash: e2701bd43ce2069cc657792f3810780a0d9ece93
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 2
# Overlapping Files: js/loader/ModuleLoaderBase.h, js/loader/ModuleLoaderBase.cpp
# Description:
#   Bug 1968890 - Part 4: Handle Dynamic import in LoadRequestedModulesRejected. r=jonco
# ==============================================================================

diff -r 360f8b5e412b -r e2701bd43ce2 js/loader/ModuleLoaderBase.h
--- a/js/loader/ModuleLoaderBase.h	Fri Oct 03 20:53:28 2025 +0000
+++ b/js/loader/ModuleLoaderBase.h	Fri Oct 03 20:53:29 2025 +0000
@@ -549,7 +549,8 @@
                                              Handle<Value> aHostDefined,
                                              Handle<Value> aError);
   static bool OnLoadRequestedModulesResolved(ModuleLoadRequest* aRequest);
-  static bool OnLoadRequestedModulesRejected(ModuleLoadRequest* aRequest,
+  static bool OnLoadRequestedModulesRejected(JSContext* aCx,
+                                             ModuleLoadRequest* aRequest,
                                              Handle<Value> aError);
 
   /**
